src/PowerShellEditorServices/Services/Symbols/Vistors/FindReferencesVisitor.cs

```diff
@@ -68,50 +68,44 @@ public override AstVisitAction VisitCommand(CommandAst commandAst)
             Ast commandNameAst = commandAst.CommandElements[0];
             string commandName = commandNameAst.Extent.Text;
 
-            if(symbolRef.SymbolType.Equals(SymbolType.Function))
+            if(_symbolRef.SymbolType.Equals(SymbolType.Function))
             {
-                if (needsAliases)
+                if (_needsAliases)
                 {
                     // Try to get the commandAst's name and aliases,
                     // if a command does not exists (if the symbol isn't an alias to a command)
                     // set command to and empty string value string command
                     // if the aliases do not exist (if the symvol isn't a command that has aliases)
                     // set aliases to an empty List<string>
-                    string command;
-                    List<string> alaises;
-                    CmdletToAliasDictionary.TryGetValue(commandName, out alaises);
-                    AliasToCmdletDictionary.TryGetValue(commandName, out command);
-                    if (alaises == null) { alaises = new List<string>(); }
+                    _cmdletToAliasDictionary.TryGetValue(commandName, out List<string> aliases);
+                    _aliasToCmdletDictionary.TryGetValue(commandName, out string command);
+                    if (aliases == null) { aliases = new List<string>(); }
                     if (command == null) { command = string.Empty; }
 
-                    if (symbolRef.SymbolType.Equals(SymbolType.Function))
+                    if (_symbolRef.SymbolType.Equals(SymbolType.Function))
                     {
                         // Check if the found symbol's name is the same as the commandAst's name OR
                         // if the symbol's name is an alias for this commandAst's name (commandAst is a cmdlet) OR
                         // if the symbol's name is the same as the commandAst's cmdlet name (commandAst is a alias)
-                        if (commandName.Equals(symbolRef.SymbolName, StringComparison.CurrentCultureIgnoreCase) ||
-                        alaises.Contains(symbolRef.ScriptRegion.Text.ToLower()) ||
-                        command.Equals(symbolRef.ScriptRegion.Text, StringComparison.CurrentCultureIgnoreCase) ||
-                        (!string.IsNullOrEmpty(command) && command.Equals(symbolRefCommandName, StringComparison.CurrentCultureIgnoreCase)))
+                        if (commandName.Equals(_symbolRef.SymbolName, StringComparison.CurrentCultureIgnoreCase)
+                            || aliases.Contains(_symbolRef.ScriptRegion.Text.ToLower())
```

Comment:
Are the elements in `aliases` always lower-case strings? If not, this may return `false` when you're expecting `true`. e.g. `new List<string>() { "Abc" }.Contains("abc")`.

Reply:
Good catch, it also needs the ignore case comparer.